Documentation/Signoffs/SensorSignoff.md

```diff
@@ -0,0 +1,94 @@
+# Sensor Subsystem Signoff
+## FUNCTION OF THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The sensor subsystem will be designed to take commands from the Microcontroller Subsystem and execute the tasks for an extended period of time. These sensors will analyze and receive data from the outside and send the new data from the interface straight to the microcontroller. Within this subsystem, the sensors will take and measure the data differently, but give the same information back for the user to obtain. Therefore, the Sensor Subsystem is an integral part of the process.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For experimental purposes, the design team has decided to get multiple types of sensors for the modular functionality to the device. 
+These types are digital, analog voltage, and analog current output sensors. While the digital sensors send bits describing the measurement of a specific parameter, the 
+analog sensors send their signals a bit differently. There are pros and cons on analog voltage or current signals in many regards. Voltage outputs for analog components
+are more common, but current signals are faster and less likely to lose current or voltage drops in the signal process. Current analog sensors have an extra step where 
+a built-in transmitter in the component takes the voltage and sets a current range for the signal to be generated. Once the current signal gets to the microcontroller, 
+it would need to go back to its voltage via a load resistor. The team plans on getting various types of sensors for the design process of this subsystem so that the user of the Air Quality Device will be able to use the modular sensor inputs for most of the sensors that the user desires.
+
+## CONSTRAINTS ON THE SUBSYSTEM
```

Comment:
Do the first two sentences contribute anything? Consider removing them.

1. Need to be specific. Exactly which items need to be measured.

3. Be more specific about the interface, in particular state that one of the listed interfaces must be supported by the sensor.

4. Remove sample rate from this number as it is covered in the next one.

5. Justify these numbers. It's not good enough to say "somebody else did it that way". Also, 2 Hz rate and 30 s period are contradictory. Too low sample rate and calibration are two different things...don't mix them up.

6. Need to specify maximum current. Power subsystem and sensor subsystem need to agree on power requirements.

7. Not sure what this is saying. Is it specifying some maximum time for the sensor to become ready to operate? If so, list the value.

Reply:
The sentences before the constraints list overviews what will be mentioned within the list. If you think it is not necessary I can delete it.

1. Every sensor to be used will measure the amount of PPM of the pollutant within it's range. For this constraint, I listed the pollutants to be measured specifically.

2. The interface is connected but not specifically an integral part of the Sensor Signoff. What do you mean by listed interfaces? The wires to be used?

3. I will remove the number of sensors to be sampled then and change to all?

4. Okay, makes sense. Should I look for another reference then? I changed the period to 50 seconds for mathematical correspondence.

5. After consulting with Caleb regarding power, I have added some more numbers within this constraint.

6. No it was more to clarify that there is a timer being implemented as a part of the subsystem functionality. This goes for every sensor to be used.